helloworld/elementary_2_3.c

```diff
@@ -0,0 +1,20 @@
+#include<stdio.h>
+
+int main() {
+
+    char name[100] ;/* define name */
+  
+  	setbuf(stdout, NULL);
+	
+	printf("Hogy hívnak? ");    
+	scanf("%s", &name);
+
+	if (name[0] == 'A' || name[0] == 'B') {
```

Comment:
❗️ a feladatban az szerepelt, hogy **"only the users Alice and Bob are greeted with their names"**.
Ez itt viszont Antoniettát és Barbarellát is köszönteni fogja, szóval még nem teljesítettük a feladat követelményét.

Reply:
Ezen még dolgozom, egyelőre abba futottam bele, hogy "Bob" == "Bob" értéke 0 :-(